docs/en/enterprise-edition/content-collections/governance/custom-build-policies/custom-build-policy-examples.adoc

```diff
@@ -71,15 +71,14 @@ If you are creating policies for Terraform, here are some examples you can use t
 [#t-basic-policy-definition]
 ==== Basic Policy Definition
 
-Use the sample guideline to create a basic policy definition using one attribute.
+In the follow example, the attribute is “tags.env”, where tags is the attribute and “env” is the key name you are searching for in a key:value pair of tags. If instead, you would like to ensure that every resource has a tag of Department:<something>, change the attribute to “tags.Department”. Note that this is case sensitive. Use the operator `equals_ignore_case` if you prefer a case insensitive check.
```

Comment:
In the following example, the attribute is `tags.env`, where `tags` is the attribute, and `env` is the key name you are searching for in a key:value pair of tags. To ensure that every resource has a `Department:<something>` tag, modify and set the attribute to `tags. Department`. Note that this is case-sensitive. Use the `equals_ignore_case` operator for a case-insensitive check.